lib/addons/fluxcd/index.ts

```diff
@@ -62,7 +86,7 @@ const defaultProps: HelmAddOnProps & FluxCDAddOnProps = {
   fluxSyncInterval: "5m0s",
   fluxTargetNamespace: "default",
   fluxPrune: true,
-  fluxTimeout: "1m"
+  fluxTimeout: "1m",

```

Comment:
remove the last ","  - i know ts does not complain, it hurts engineers across the world

Reply:
Fixed.